src/third-parties/google-analytics/data.json

```diff
@@ -11,8 +11,7 @@
       "action": "append"
     },
     {
-      "code": "window.dataLayer=window.dataLayer||[];window.gtag=function gtag(){window.dataLayer.push(arguments);};gtag('js',new Date());gtag('config','${args.id}')",
-      "strategy": "worker",
+      "code": "window.dataLayer=window.dataLayer||[];window.gtag=function gtag(){window.dataLayer.push(arguments);};gtag('js',new Date());gtag('config','{{id}}')",      "strategy": "worker",
```

Comment:
Nit: Identation

I thought I already set up Prettier in this package, but maybe I need to take another look 🤔

Reply:
🤦‍♂️ - fixed